docs/Software Architecture Document/View-to-View Relations/index.md

```diff
@@ -21,6 +21,7 @@ The following table is a mapping between the elements in the Component view and
 | ***Connection Handler***  |                       | X                  |                     |              |
 | ***Security Manager***    |                       | X                  |                     |              |
 | ***HMI Message Handler*** |                       |                    | X                   | X            |
+| ***Plugin Manager***      | X                     |                    |                     |              |
```

Comment:
@LuxoftAKutsan, 
But it's still SDL component described in the component view. And it's related to one of the existing layers.